collective/googlenews/browser/views.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+from collective.googlenews.config import PLONE_VERSION
+from collective.googlenews.interfaces import (
+    IGoogleNewsEditorFieldsSettings
+)
+from plone import api
+from plone.formwidget.namedfile.converter import b64decode_file
+from plone.namedfile.browser import Download
+from plone.namedfile.file import NamedImage
+from Products.Five import BrowserView
+from zExceptions import NotFound
+from zope.component import getMultiAdapter
+from zope.component import queryAdapter
+
+
+if PLONE_VERSION < '4.3':
+    from collective.syndication.interfaces import IFeed
+    from collective.syndication.interfaces import IFeedSettings
+else:
+    from Products.CMFPlone.interfaces.syndication import IFeed
+    from Products.CMFPlone.interfaces.syndication import IFeedSettings
+
+
+class AtomFeedView(BrowserView):
```

Comment:
@rodfersou can we just inherit from the original adapter and change just the logo?